lambda/movieFinder.js

```diff
@@ -11,7 +13,17 @@ var find = function(slots, sessionAttributes) {
         validateSlots(queryInfo)
         .then(() => findMovieFromSources(queryInfo))
         .then((movieLists) => processMovieLists(movieLists))
-        .then((singleMovieList) => resolve(singleMovieList))
+        .then((singleMovieList) => {
+            if (singleMovieList.length == 0 ) {
+                var err = {
+                    type: "NotFound"
+                }
+                
+                reject(err)
```

Comment:
I was thinking of returning an error object. https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Error
We can add the message inside the error object